src/software/ai/evaluation/calc_best_shot.cpp

```diff
@@ -9,23 +9,29 @@ namespace Evaluation
                                            const std::vector<Point> &obstacles,
                                            double radius)
```

Comment:
Just leaving a blocking comment here to remove the `src/core` binary file.

Reply:
I removed it and added a new line to gitignore